CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Breaking
```

Comment:
Shall we start an upgrade guide ?

Reply:
I thought about it, but decided that trying to manage merge conflicts for the upgrade guide would be annoying. So as of now I was planning to do it at the end 🤷‍♂️